src/bin/chessmarkable/main.rs

```diff
@@ -25,45 +26,56 @@ use std::thread::sleep;
 use std::time::{Duration, SystemTime};
 
 #[derive(Clap)]
-#[clap(version = crate_version!(), author = crate_authors!())]
+#[clap(version = crate_version ! (), author = crate_authors ! ())]
 pub struct Opts {
     #[clap(
-        long,
-        short = 'X',
```

Comment:
Sorry about this must have been intellij auto-formatting

Reply:
I don't think that this is a problem. I did not use rustfmt for this project since I wanted some longer lines for it to be more readable. But I should really have just supplied an according rustfmt.toml file (which IntelliJ would probably use as well) to properly fix this.

Reply:
Ah I see, some people mind other people messing with their formatting in their commits so I usually try to undo any automatic changes intellij made during my code review. But if you don't mind this I'll let it be